packages/docusaurus-theme-translations/locales/et/theme-common.json

```
@@ -0,0 +1,74 @@
+  "theme.TOCCollapsible.toggleButtonLabel": "Selle lehe peal",
+  "theme.admonition.caution": "hoiatus",
+  "theme.admonition.danger": "oht",
+  "theme.admonition.info": "info",
+  "theme.admonition.note": "teade",
+  "theme.admonition.tip": "nõuanne",
+  "theme.admonition.warning": "hoiatus",
+  "theme.blog.archive.description": "Arhiiv",
+  "theme.blog.archive.title": "Arhiiv",
+  "theme.blog.paginator.navAriaLabel": "Blogi lehekülje navigatsioon",
+  "theme.blog.paginator.newerEntries": "Uuemad sissekanded",
+  "theme.blog.paginator.olderEntries": "Vanemad sissekanded",
+  "theme.blog.post.paginator.navAriaLabel": "Blogi postituste navigatsioon",
+  "theme.blog.post.paginator.newerPost": "Uuemad postitused",
+  "theme.blog.post.paginator.olderPost": "Vanemad postitused",
+  "theme.blog.post.plurals": "Üks postitus|{count} postitust",
+  "theme.blog.post.readMore": "Loe edasi",
+  "theme.blog.post.readMoreLabel": "Loe edasi {title} kohta",
+  "theme.blog.post.readingTime.plurals": "Ühe min lugemine|{readingTime} min lugemine",
+  "theme.blog.sidebar.navAriaLabel": "Blogi hiljutiste postituste navigatsioon",
+  "theme.blog.tagTitle": "{nPosts} märgistatud \"{tagName}\"",
+  "theme.colorToggle.ariaLabel": "Vaheta heleda ja tumeda teema vahel (currently {mode})",
+  "theme.colorToggle.ariaLabel.mode.dark": "tume teema",
+  "theme.colorToggle.ariaLabel.mode.light": "hele teema",
+  "theme.common.editThisPage": "Redigeeri seda lehte",
+  "theme.common.headingLinkTitle": "Link {heading}",
+  "theme.common.skipToMainContent": "Liigu peamise sisu juurde",
+  "theme.docs.DocCard.categoryDescription.plurals": "1 ese|{count} eset",
+  "theme.docs.breadcrumbs.home": "Koduleht",
+  "theme.docs.breadcrumbs.navAriaLabel": "Breadcrumbs",
+  "theme.docs.paginator.navAriaLabel": "Dokumentatsiooni lehed",
+  "theme.docs.paginator.next": "Järgmine",
+  "theme.docs.paginator.previous": "Eelmine",
+  "theme.docs.sidebar.closeSidebarButtonAriaLabel": "Sulge navigatsiooniriba",
+  "theme.docs.sidebar.collapseButtonAriaLabel": "Sulge külgriba",
+  "theme.docs.sidebar.collapseButtonTitle": "Sulge külgriba",
+  "theme.docs.sidebar.expandButtonAriaLabel": "Ava külgriba",
+  "theme.docs.sidebar.expandButtonTitle": "Ava külgriba",
+  "theme.docs.sidebar.navAriaLabel": "Dokumentatsiooni külgriba",
+  "theme.docs.sidebar.toggleSidebarButtonAriaLabel": "Lülita navigatsiooniriba",
+  "theme.docs.tagDocListPageTitle": "{nDocsTagged} koos \"{tagName}\"",
+  "theme.docs.tagDocListPageTitle.nDocsTagged": "Üks dokument märgistatud|{count} dokumenti märgistatud",
+  "theme.docs.versionBadge.label": "Versioon: {versionLabel}",
+  "theme.docs.versions.latestVersionLinkLabel": "viimane versioon",
+  "theme.docs.versions.latestVersionSuggestionLabel": "Uuendatud dokumentatsiooni lugemiseks vaata {latestVersionLink} ({versionLabel}).",
+  "theme.docs.versions.unmaintainedVersionLabel": "See on  {siteTitle} {versionLabel} dokumentatsioon, mida enam ei täiendata.",
```

```suggestion
  "theme.docs.versions.unmaintainedVersionLabel": "See on {siteTitle} {versionLabel} dokumentatsioon, mida enam ei täiendata.",
```

I assume the double space is a typo here, fixing it.